<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClusteringView</class>
 <widget class="QWidget" name="ClusteringView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frmClustering">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="btnSegment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start clustering</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblClustering">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clustering</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSegInputFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Target file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="FileChooser" name="wgtSegFileChooser" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="FilterOptionView" name="wgtFilterOptionView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>327</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Filter</string>
     </attribute>
    </widget>
    <widget class="MinCutOptionView" name="wgtMinCutOptionView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>327</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Min cut segmentation</string>
     </attribute>
    </widget>
    <widget class="SegmentationOptionView" name="wgtSegOptionView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>327</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Segmentation</string>
     </attribute>
    </widget>
    <widget class="ClusteringOptionView" name="wgtClusterOptionView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>327</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Clustering</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="lblms">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>590</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdCompTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>590</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblCompTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Computation time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>toolBox</zorder>
   <zorder>btnSegment</zorder>
   <zorder>lblClustering</zorder>
   <zorder>lblSegInputFile</zorder>
   <zorder>wgtSegFileChooser</zorder>
   <zorder>lblms</zorder>
   <zorder>lcdCompTime</zorder>
   <zorder>lblCompTime</zorder>
  </widget>
  <widget class="QFrame" name="frmCtrlModel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>691</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblOutputFile">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="FileChooser" name="wgtOutFileChooser" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrevModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextModel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveModel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="PCLViewer" name="wgtPCLViewer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>671</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblExtrModelsPreview">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Extracted models preview</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNrModelPoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>69</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>#cloud points</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNrCloudsPoints">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNrKeyPoints">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>71</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNrKeypoints">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>#keypoints</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PCLViewer</class>
   <extends>QWidget</extends>
   <header location="global">pclviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileChooser</class>
   <extends>QWidget</extends>
   <header location="global">filechooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterOptionView</class>
   <extends>QWidget</extends>
   <header location="global">filteroptionview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SegmentationOptionView</class>
   <extends>QWidget</extends>
   <header location="global">segmentationoptionview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClusteringOptionView</class>
   <extends>QWidget</extends>
   <header location="global">clusteringoptionview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinCutOptionView</class>
   <extends>QWidget</extends>
   <header location="global">mincutoptionview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
